.github/workflows/pull_request_clean_up.yml

```diff
@@ -0,0 +1,30 @@
+name: Clean-Up GitHub Pages
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  delete_pull_request_preview:
+    runs-on: ubuntu-latest
+    env:
+      PR_PATH: pull/${{github.event.number}}
+    steps:
+      - name: Create empty dir to clear PR preview
+        run: mkdir empty
+
+      - name: Deploy empty dir to GitHub Pages PR preview URL
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./empty
+          destination_dir: ${{ env.PR_PATH }}
```

Comment:
What's the motivation for deleting the meta-site directory for the PR when it's closed? I can see why we'd want to remove it from the index (might be nice to have a second index of closed PRs) but it's not like we're saving any space in the repo by doing this since the data is still present in the old commits, we're just making it no longer visible. I feel like we might as well just keep the pages around so that the links still work, kind of like how Vercel keeps all of our previous deploy previews alive at a permalink.

Reply:
I just figured we would want to keep things clean-ish on the `gh-pages` branch, and it has the additional value of updating the meta-site's PR index since that's pulling from the `gh-pages` branch contents rather than actually querying for open PRs.

That said, it should be easy enough to query the GitHub API for open PRs and create links based on that instead, and then you're right that we don't have to worry about doing any clean-up. Let me see if I can knock that out as a quick win.

Reply:
Got this working relatively easily and then spent way too much time playing with emojis and now my eyes are bleeding from what I've wrought: https://la-ldaf.github.io/ldaf-site/pull/

If we don't get our CMS set up soon we're going to end up migrating the meta site into Svelte as well 😅 

Anyway now that this is working, I'll remove the Clean Up action and update the PR description.